src/tasks/JointTask.cpp

```diff
@@ -15,15 +15,16 @@ JointTask::JointTask(std::shared_ptr<Sai2Model::Sai2Model>& robot,
 					 const std::string& task_name, const double loop_timestep)
 	: TemplateTask(robot, task_name, TaskType::JOINT_TASK, loop_timestep) {
 	// selection for full joint task
-	_joint_selection = MatrixXd::Identity(getConstRobotModel()->dof(), getConstRobotModel()->dof());
+	_joint_selection = MatrixXd::Identity(getConstRobotModel()->dof(),
```

Comment:
It looks like all the changes in JointTask are not from you, but you need to merge/rebase on master